rllib/evaluation/env_runner_v2.py

```diff
@@ -55,15 +55,35 @@
 class _PerfStats:
     """Sampler perf stats that will be included in rollout metrics."""
 
-    def __init__(self):
+    def __init__(self, ema=False, ema_coeff=0.001):
```

Comment:
instead of both a flag and a coefficient value us ema_coef = None to indicate no ema. Also specify the type in the signature:
```
__init__(self, ema_coef: Optional[float] = None):
### documentation
```

Reply:
Also please document where the coef is (1-alpha) * old. + alpha * new or alpha * old + (1 - alpha) * new

Reply:
yeah, good idea. this is cleaner.
I was trying not to let users specify a number. but since this needs to be turned on, it should be fine.